### PR TITLE
swap all use of get_event_triggers to rx.EventHandler props

### PR DESCRIPTION
**Pull request by Lendemor**
what the title says.

### CODE REVIEW

File: reflex/components/datadisplay/dataeditor.py [hunk 1]
```diff
@@ -223,35 +276,6 @@ def _get_imports(self):
             },
         )
 
-    def get_event_triggers(self) -> Dict[str, Callable]:
```
Comment (masenf) [inline reflex/components/datadisplay/dataeditor.py:226]: removing this has resulted in the addition of the base Component.get_event_triggers default triggers, is that appropriate for this component? or should we have a `get_event_triggers` here that returns an empty dict?

File: reflex/components/radix/themes/components/alert_dialog.py [hunk 2]
```diff
@@ -48,18 +40,14 @@ class AlertDialogContent(elements.Div, RadixThemesComponent):
...
-            EventTriggers.ON_ESCAPE_KEY_DOWN: lambda e0: [e0],
-        }
+    # Fired when the dialog is opened.
+    on_open_auto_focus: EventHandler[lambda e0: [e0]]
+
+    # Fired when the dialog is closed.
+    on_close_auto_focus: EventHandler[lambda e0: [e0]]
+
+    # Fired when the escape key is pressed.
+    on_escape_key_down: EventHandler[lambda e0: [e0]]
```
Comment (masenf) [inline reflex/components/radix/themes/components/alert_dialog.py:50]: i wonder if we should mention that the intended purpose of these event handlers is primarily to call `rx.prevent_default`

**Review by masenf — DISMISSED** [review]

**Review by picklelo — DISMISSED** [review]
Can we delete the `EventTriggers` class now?

**Comment by Lendemor** [comment]
> Can we delete the `EventTriggers` class now?

I'm not sure yet, they are still used in the base Component I think.

I'm hesitant to convert this one to rx.EventHandler right now because I'm not sure it won't have side effect.

**Review by picklelo — APPROVED** [review]

File: reflex/components/datadisplay/dataeditor.py (hunk 1 above)
Comment (Lendemor) [reply reflex/components/datadisplay/dataeditor.py]: Adding `get_event_triggers` that returns `{}` has caused events triggers to be completely removed from the docs.

**Pull request merged**